concepticondata/conceptrelations.tsv

```diff
@@ -770,3 +770,12 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3805	SHY OR ASHAMED	narrower	1894	ASHAMED
 3809	SCREAM (PRODUCE A CRY)	producedby	2522	SCREAM (NOISE)
 298	SHRIEK	similar	3809	SCREAM (PRODUCE A CRY)
+3213	HIRE OR RENT	narrower	3831	RENT (SOMETHING)
```

Comment:
We would also need to add a relation for 856 HIRE, right?

Reply:
Thanks for leaving a review! That relation (HIRE OR RENT narrower HIRE) already exists in l. 623.

Reply:
Ah yes!